src/services/iotHubManagerService.js

```diff
@@ -15,4 +15,9 @@ export class IoTHubManagerService {
     return HttpClient.get(`${ENDPOINT}devices?query=${query}`)
       .map(toDevicesModel);
   }
+
+  /** Deletes a device */
+  static deleteDevice(id) {
+    return HttpClient.delete(`${ENDPOINT}devices/${id}`);
```

Comment:
Let's add a map here to create a synthetic response object.

```
return HttpClient.delete(`${ENDPOINT}devices/${id}`)
    .map(() => ({ deletedDeviceId: id }))
```

This will simplify the code in the flyout an other locations where we may need to delete a device.